src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+        this.color = ColorSupplier.colorGen();
+    }
+
+    public Circle() {
+        this.radius = FigureSupplier.lengthGen();
+        this.color = ColorSupplier.colorGen();
+    }
```

Comment:
Let's delete this constructor, same in all places

Reply:
so leave it without default constructor?